instana/tracer.py

```diff
@@ -109,15 +121,15 @@ def start_span(self,
 
         return span
 
-    def inject(self, span_context, format, carrier):
+    def inject(self, span_context, format, carrier, disable_w3c_trace_context=True):
```

Comment:
With `disable_w3c_trace_context` being `True` by default services that were instrumented manually would need to make changes in order to support W3C trace context. Do you see any issues with using this context by default, or we can make it opt-out instead of opt-in?